projects/2019_12_16_MultiSig_QT_Wallet.md

```diff
@@ -0,0 +1,90 @@
+* Higher security & control, backup feature, and joint-account capability, provided by the multi-sig-transaction accessibility.
+* Increase the density of the PKT Network nodes, to contribute to building the infrastructure of a functional decentralized network.
+
+In the list of examples of [Projects Supported by The Network Steward](http://github.com/pkt-cash/ns-projects/blob/master/readme.md), the first example is a wallet. As of the date of preparation of this proposal, the only available wallet is a [text-based wallet](http://github.com/pkt-cash/pktwallet).
+
+## 2. Project Deliverables
+The project will create new software for a standalone application, which should be downloaded and installed from an Installer. The installation will come bundled and ready to run a full-node, [PKTD](https://github.com/pkt-cash/pktd), as well as a [PKT Wallet](http://github.com/pkt-cash/pktwallet) instance.
+
+This deliverables are:
+* New software for a standalone application.
+* The software will be available under the [ISC](https://opensource.org/licenses/ISC) license.
+* Any artistic non-software assets will be available under the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
+* The maintainer of this software will be: Vishnu Seesahai
+* The software will be hosted on https://github.com/pkt-cash, if the network steward wishes it so.
+
+### 2.1 Development overview
+The development will be carried out by using the QT Software toolkit due to its history of reliability, portability, and low memory usage.
+
+The wallet will have a graphical user interface (GUI).
+Compatible operating systems: MacOS, Unix.
+The software will have future interoperability with Lightning Network.
+
+For the avoidance of doubt, the development will not be an SPV wallet, and it will not have features for colored coins. This software will not have a low-resource footprint because it will run a full node.
+
+### 2.2 Wallet target features
+1. Send and receive transactions.
+2. Send transactions to multiple addresses at once.
+3. Batch transactions for large UTXO input sets.
+4. Sign transactions.
+5. Verify transactions.
+6. Generate new regular addresses and new multi-sig addresses.
+7. Export private keys and import new private keys.
+8. Encrypt and decrypt messages.
+9. Password-protect the wallet.
+10. Save and restore the wallet backup from seed.
+
+### 2.3 Wireframe
+The following image is a wireframe of the application for illustration purposes. The design and location of the features may differ in the final version.
+<div style="max-width:1000px"> <media-tag src="https://files.cryptpad.fr/blob/49/491527e3bbf27b61ba591fa9d6b55df7fec38a010e2fbd38" data-crypto-key="cryptpad:k9IvGYHKMNLw1wxXPFQEIYuqTuu3UqDKKx9NsLUn32g="></media-tag> </div>
```

Comment:
This doesn't seem to render.

Reply:
That was a direct copy from CryptPad and CryptPad uses encrypted images which don't render outside. Anyway since I reviewed the pad I have the drawing here: https://cryptpad.fr/file/#/2/file/i7jC-Hb5h9xoLKr8fTnhZULF/

Reply:
@artrepreneur it would be a good idea to put the picture somewhere such as imgur.com and then add a link to that here since then it will appear for the future.